### PR TITLE
2023 Paketo TSC election admin

### PR DESCRIPTION
**Pull request by christopherclark**

### CODE REVIEW

**Comment by christopherclark** [comment]
@emalm @dmikusa I think this should be good to go. I updated the issue templates so those links look to be working correctly.  Once the [Paketo community repository]( https://github.com/paketo-buildpacks/community) is updated, we may want to update the README to point to pertinent sections.

File: elections/2023/Paketo-TSC/election_desc.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The role of this election is to fill the two (2) seats due for election this year on the Paketo Buildpacks TSC. The elected members will serve two (2) year terms. This is a brief summary of the election; for more details, such as how to become a candidate, [see the README](https://github.com/cloudfoundry/community/tree/main/elections/2023/Paketo-TSC#readme).
+
+Eligibility
+Any individual who has contributed to Paketo Buildpacks in the twelve months prior to the election is eligible to vote in the Steering Committee election.. You will see either ELIGIBLE or NOT ELIGIBLE at the top of this screen. If Elekto fails to recognize you as eligible to vote in this election even though you believe you should be, please file a voting exception request [within the Elekto app](https://elections.cloudfoundry.org/app/elections/2023---Paketo-TSC/exception) or via an issue on the community repository. The deadline for filing exceptions is June 14th, 11:59 pm PT.
```
Comment (dmikusa) [inline elections/2023/Paketo-TSC/election_desc.md:4]:
```suggestion
Any individual who has contributed to Paketo Buildpacks in the twelve months prior to the election is eligible to vote in the Steering Committee election. You will see either ELIGIBLE or NOT ELIGIBLE at the top of this screen. If Elekto fails to recognize you as eligible to vote in this election even though you believe you should be, please file a voting exception request [within the Elekto app](https://elections.cloudfoundry.org/app/elections/2023---Paketo-TSC/exception) or via an issue on the community repository. The deadline for filing exceptions is June 14th, 11:59 pm PT.
```

**Review by dmikusa — APPROVED** [review]

File: elections/2023/Paketo-TSC/election.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: 2023 Paketo Buildpacks Technical Steering Committee election
+organization: Cloud Foundry Foundation
+start_datetime: 2023-06-01 06:59:00 # 11:59 pm PDT, 02 Jun 2022
+end_datetime: 2023-06-16 06:59:00 # 11:59 pm PDT, 17 Jun 2022
```
Comment (emalm) [inline elections/2023/Paketo-TSC/election.yaml:4]:
```suggestion
start_datetime: 2023-06-02 06:59:00 # 11:59 pm PDT, 01 Jun 2023
end_datetime: 2023-06-17 06:59:00 # 11:59 pm PDT, 16 Jun 2023
```

File: elections/2023/Paketo-TSC/election.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- github
+- slack
+election_officers:
+- christopherclark
+- emalm
+- dmikusa
+- sgreenberg
+eligibility: Any individual who has contributed to Paketo Buildpacks in the twelve months prior to the election is eligible to vote in the Steering Committee election. Each election cycle, an initial set of voters will be identified through automated reporting. Any individual who has at least 10 measurable contributions in the last 12 months will be automatically added to the eligible voter list.
+exception_description: If you feel you are eligible to vote but are not represented in the list of voters, please request an exception to allow you to vote via the Elekto application.
+exceptions_due: 2023-06-14 06:59:00 # 11:59 pm PT, 14 Jun 2023
```
Comment (emalm) [inline elections/2023/Paketo-TSC/election.yaml:19]:
```suggestion
exceptions_due: 2023-06-14 06:59:00 # 11:59 pm PT, 13 Jun 2023
```

**Review by emalm — APPROVED** [review]

**Pull request merged**